pyproject.toml

```diff
@@ -543,7 +543,8 @@ markers = [
     "integration: mark test as an integration test.",
     "slow: mark tests taking longer than 1 second.",
     "unit: mark a test as a unit test.",
-    "clickhouse: mark a test as Clickhouse-dependent.", 
+    "aws: test uses an AWS service (sqs, sns, s3 etc).",
```

Comment:
```suggestion
    "aws: test relates to an AWS service (sqs, sns, s3 etc).",
```

Reply:
I did `mark all tests that require aws credentials` but 🤷 .